src/gui/transferlistmodel.cpp

```diff
@@ -632,9 +632,12 @@ bool TransferListModel::setData(const QModelIndex &index, const QVariant &value,
 
 void TransferListModel::addTorrents(const QVector<BitTorrent::Torrent *> &torrents)
 {
-    int row = m_torrentList.size();
-    beginInsertRows({}, row, (row + torrents.size()));
+    qsizetype row = m_torrentList.size();
```

Comment:
I would use `auto` in such cases (since exact type has no meaning in this context), but of course you won't agree with me.

Reply:
I need it to induce the type of `total`.

Reply:
> I need it to induce the type of `total`.

Won't `const auto total = row + torrents.size();` work?

Reply:
I mean "I don't care about exact type of size and I just use this type for `row` and `total`".

Reply:
I still don't like it. What if someone later changed `m_torrentList` to std container and then `row` and `total` would become unsigned type which violates coding style? This would take forever to be discovered.
Moreover, what if later someone inferred `m_torrentList` type wrong (he thought it was qt container, but it is actually std) and performed some dangerous operations (for example, subtractions) on variable from `size()` and results in integer underflow?

Those are hypothetical examples but would rather stop those from happening than giving them a chance.